shapely/_geometry_helpers.pyx

```diff
@@ -384,3 +385,11 @@ def collections_1d(object geometries, object indices, int geometry_type = 7, obj
             geom_idx_1 += collection_size[coll_idx]
 
     return out
+
+
+def _geom_factory(uintptr_t g):
+
+    with get_geos_handle() as geos_handle:
+        geom = PyGEOS_CreateGeometry(<GEOSGeometry *>g, geos_handle)
```

Comment:
We probably should rename the `PyGEOS_*` Cython helpers to `Shapely_*` (another PR).

Reply:
> We probably should rename the `PyGEOS_*` Cython helpers to `Shapely_*` (another PR).

For some that might indeed make sense, for others where it's a Python C wrapper ("Py") around something GEOS related, it could also makes sense to keep it (but didn't check the different cases in detail)